lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -78,6 +107,11 @@ export class RefreshTokenClient extends BaseClient {
         return this.acquireToken(refreshTokenRequest);
     }
 
+    /**
+     *
+     * @param request
+     * @param authority
+     */
```

Comment:
nit: Adds no additional information, and instead clutters the code. If you add comments to methods, consider filling them in with info.

Reply:
This is a miss. Will add now. thanks for catching it

Reply:
Done